proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -85,6 +86,8 @@ message Experiment {
   string notes = 14;
   // Associated job's id.
   string job_id = 15;
+  // Original id of a forked or continued experiment.
+  google.protobuf.Int32Value forked_from = 16;
```

Comment:
question: why does this need to be a `google.protobuf.Int32Value` and not just an `int32`?

Reply:
I prefer the `google.protobuf.` type because it can be `null` if it's empty/blank, if it's int32 it would send a 0